src/Defaults.php

```diff
@@ -2,7 +2,7 @@
 namespace Ably;
 
 class Defaults {
-    const API_VERSION = '1.1';
+    const API_VERSION = '2.0';
```

Comment:
Is it protocol version? Should it be `2` then?

Reply:
Let me check !

Reply:
Yeah, it should be `2`. Seems I followed https://github.com/ably/specification/pull/88/files.

Reply:
Silly mistake !

Reply:
updated